ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/mac-security.j2

```diff
@@ -23,11 +23,20 @@ mac security
       {{ key_cli }}
 {%             endif %}
 {%         endfor %}
+{%         if profile.mka.key_server_priority is arista.avd.defined %}
+      mka session key-server priority {{ profile.mka.key_server_priority }}
+{%         endif %}
 {%         if profile.mka.session.rekey_period is arista.avd.defined %}
       mka session rekey-period {{ profile.mka.session.rekey_period }}
 {%         endif %}
 {%         if profile.sci is arista.avd.defined(true) %}
       sci
 {%         endif %}
+{%         for protocol in profile.l2_protocols | arista.avd.default([]) %}
```

Comment:
please use natural_sort on "name" here. It will automatically return an empty list if the input var was not defined.

Reply:
If they accept different options, then we should indeed change it to a dict of known protocols. Thank you for catching this.

Reply:
> Collaborator

I am having trouble with the `natural_sort` filter:

```
ok: [load-interval -> localhost]
fatal: [mac-security-eth-po-entropy -> localhost]: FAILED! => {"msg": "'l2_protocols' is undefined"}
ok: [loopbacks-interfaces -> localhost]
```

molecule complains that `l2_protocols` is undefined, but since I use the natural_sort filter, this should not happen:

```jinja
{%         for l2_protocol in profile.l2_protocols | arista.avd.natural_sort %}
      l2-protocol {{ l2_protocol }} {{ l2_protocols[l2_protocol].mode }}
{%         endfor %}
{%     endfor %}
```

Reply:
Thanks, I've fixed the data structure and templates. Should look ok now